### PR TITLE
Varnish syntax prefers double to single quote marks

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

**Comment by stevepiercy** [comment]
Before merging, I could do the same to the old Varnish template, too.

File: roles/varnish/templates/default.vcl4.j2 [hunk 1]
```diff
@@ -64,7 +64,7 @@ sub vcl_recv {
 {%if proxy_cache_block_basic_auth|default(True) %}
     # Don't allow basic auth via this route.
     # If you want to login at the Zope root, use an ssh tunnel.
-    if (req.http.Authorization && req.http.Authorization !~ 'Bearer') {
+    if (req.http.Authorization && req.http.Authorization !~ "Bearer") {
```
Comment (fulv) [inline roles/varnish/templates/default.vcl4.j2]: I actually ran into this, too.   Sorry for forgetting to update the repo!

**Comment by fulv** [comment]
I don't know if anybody is still using vcl < 4.

**Comment by stevepiercy** [comment]
Varnish 4.0, released Apr 10, 2014, is deprecated.

Should we drop support for Varnish < 4?

I vote yes. If after removing support, someone reports an issue about its removal, it's easy enough to restore.

**Comment by fulv** [comment]
+1  for removing Varnish < 4 support

**Pull request merged**